### PR TITLE
unify capitalization of variables and parameters

### PR DESCRIPTION
**Pull request by fbenke-pik**
## Purpose of this PR

Unifies capitalization of variables an parameters so that REMIND passes soon-to-be introduced [codeCheck](https://github.com/pik-piam/gms/pull/84)

Whenever possible, the code uses the capitalization as declared in declarations.gms and sets.gms.

## Type of change

- [x] Refactoring

## Checklist:

- [x] My code follows the [coding etiquette](https://github.com/remindmodel/remind/blob/develop/main.gms#L80)
- [x] I performed a self-review of my own code
- [x] I explained my changes within the PR, particularly in hard-to-understand areas
- [x] I checked that the [in-code documentation](https://github.com/remindmodel/remind/blob/develop/main.gms#L120) is up-to-date
- [x] I adjusted the reporting in [`remind2`](https://github.com/pik-piam/remind2) where it was needed
- [x] All automated model tests pass (`FAIL 0` in the output of `make test`)

### CODE REVIEW

File: core/bounds.gms [hunk 1]
```diff
@@ -28,8 +28,8 @@ vm_costTeCapital.fx(t,regi,teNoLearn)     = pm_inco0_t(t,regi,teNoLearn);
 loop(pe2se(enty,enty2,te)$((not sameas(te,"biotr"))  AND (not sameas(te,"biodiesel")) AND (not sameas(te,"bioeths")) AND (not sameas(te,"gasftcrec")) AND (not sameas(te,"gasftrec"))
 AND (not sameas(te,"tnrs"))),
   vm_cap.lo(t,regi,te,"1")$(t.val gt 2026 AND t.val le 2070) = 1e-7;
-  if( (NOT teccs(te)), 
-    vm_deltacap.lo(t,regi,te,"1")$(t.val gt 2026 AND t.val le 2070) = 1e-8;
+  if( (NOT teCCS(te)),
```
Comment (fbenke-pik) [inline core/bounds.gms:31]: test

**Review by LaviniaBaumstark — COMMENTED** [review]
did you also check the scripts part of teh GAMS repo and that the reporting is still working?

**Review by 0UmfHxcvx5J7JoaOhFSs5mncnisTJJ6q — APPROVED** [review]
… adding insult to injury asking me to approve this.

**Comment by fbenke-pik** [comment]
Nah, I added you to give you the chance to raise concerns before this is merged (also, because it affects the industry module a lot).

**Pull request merged**